src/app/dashboard/notifications/page.tsx

```diff
@@ -0,0 +1,28 @@
+"use client"
+import NotificationListing from "@/app/components/NotificationListing";
+import { useNotifications } from "../../../../ctx/NotificationsContext";
+
+export default function Page() {
+  const { notifications } = useNotifications();
+
+  return (
+    <>
+      <div className="flex flex-col w-full h-full gap-y-4 bg-white">
+        {notifications.map((notification) => {
+          return (
+            <NotificationListing
+              key={notification.id}
+              //   title={notification.title}
+              //   description={notification.description}
+              //   createdAt={notification.createdAt}
+              //   doctorId={notification.doctorId}
+              //   id={id}
+              //   patientId={notification.}
```

Comment:
Remove this codes if you don't need them !!

Reply:
done